### PR TITLE
50 kubernetes

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build the kubernetes page

## QA
- Go to /kubernetes
- Check the content matches [the copy doc](https://docs.google.com/document/d/1t-fQU3HfRNRVoIRjsFJRyDbHq-NvP7YapdnijAqBtNI/edit#heading=h.yjalla1b1o96)

## Issue / Card
Fixes https://github.com/canonical-websites/jp.ubuntu.com/issues/50

### CODE REVIEW

**Comment by kwm14** [comment]
## Design review

Looking good @anthonydillon just a couple of design updates 😉 

**パブリッククラウドとオンプレミス**
- [ ] Remove 'Oracle Cloud' logo as it's not featured on copy doc
- [ ] Remove 'Cisco' logo as it's not featured on copy doc

**Kubernetesエンタープライズ向けサポート**
- [ ] Missing link on 'Ubuntu Advantage' inside intro copy. Link here: https://www.ubuntu.com/support

**Comment by deadlight** [comment]
@anthonydillon you're proposing this to master, not v2

**Comment by anthonydillon** [comment]
@deadlight @kwm14 Addressed your comments.

File: templates/kubernetes.html [hunk 1]
```diff
@@ -0,0 +1,570 @@
+{% extends "_base/base.html" %}
+
+{% block title %}人工知能/機械学習 | Ubuntu{% endblock %}
```
Comment (deadlight) [inline templates/kubernetes.html]: Needs updating - the page title is probably too long so maybe:
```suggestion
{% block title %}Kubernetes | Ubuntu{% endblock %}
```

File: templates/kubernetes.html [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light is-bordered" id="bare-metal">
+    <div class="row">
+      <div class="col-8">
```
Comment (deadlight) [inline templates/kubernetes.html]: Maybe remove this div to stop the single-character wrapping in the heading

**Comment by kwm14** [comment]
Thanks @anthonydillon Design updates LGTM 👍

File: templates/kubernetes.html [hunk 3]
```diff
@@ -0,0 +1,570 @@
...
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip--light is-bordered">
+    <div class="row">
+      <div class="col-12">
+        <h3>DiscovererおよびDiscoverer Plus向けのAI（人工知能）/ ML（機械学習）アドオン</h3>
+        <p>機械学習のツールと運用を幅広く理解するためにK8sワークショップを活用してください。開発者ワークステーションからデータセンターやパブリッククラウドまで、完全なAI分析パイプラインの構築が可能です。GPGPUの統合、パブリッククラウドの最適化、KubeflowやTensorflowなどのツールが対象です。ワークショップには独立系のデータサイエンティストによる評価が含まれています。Canonicalはハードウェア、クラウド、データ分析の大手企業との協力により、お客様のAI戦略への統合のための幅広い選択肢を提供しています。</p>
+        <p><a href="/ai">KubernetesとAI/MLに関する詳細&nbsp;&rsaquo;</a></p>
```
Comment (deadlight) [inline templates/kubernetes.html]: This link was wrong in the copydoc, I've updated it.
```suggestion
        <p><a href="/ai-ml">KubernetesとAI/MLに関する詳細&nbsp;&rsaquo;</a></p>
```

File: templates/kubernetes.html [hunk 4]
```diff
@@ -0,0 +1,570 @@
...
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12" style="overflow-x: auto;">
+        <table class="p-table" style="width: auto;">
+          <thead>
+            <tr>
+              <th>&nbsp;</th>
+              <th class="u-align--center">Ubuntu</th>
+              <th class="p-table__cell--highlight u-align--center" colspan="2">Kubernetes向けUbuntu Advantage</th>
```
Comment (deadlight) [inline templates/kubernetes.html]: These cells should be highlighted (p-table__cell--highlight). If you rebase from v2, you will get the latest local styling.

**Comment by anthonydillon** [comment]
@deadlight all done

File: templates/kubernetes.html (hunk 2 above)
Comment (deadlight) [reply templates/kubernetes.html]: @anthonydillon Can you try removing the col-8? I think that it looks so much better.

**Pull request merged**